FunctionSignatureMap.php

```diff
@@ -1647,7 +1647,7 @@
 'date_timestamp_set' => ['DateTime|false', 'object'=>'DateTime', 'unixtimestamp'=>'int'],
 'date_timezone_get' => ['DateTimeZone', 'obj'=>'DateTimeInterface'],
 'date_timezone_set' => ['DateTime|false', 'object'=>'DateTime', 'timezone'=>'DateTimeZone'],
-'datefmt_create' => ['IntlDateFormatter', 'locale'=>'string', 'datetype'=>'int', 'timetype'=>'int', 'timezone='=>'int', 'calendar='=>'int|IntlCalendar', 'pattern='=>'string'],
+'datefmt_create' => ['IntlDateFormatter', 'locale'=>'string', 'datetype'=>'int', 'timetype'=>'int', 'timezone='=>'string|null', 'calendar='=>'int|IntlCalendar|null', 'pattern='=>'string'],
```

Comment:
https://www.php.net/manual/en/intldateformatter.create.php

> This can also be an IntlTimeZone or a DateTimeZone object.

IntlDateFormatter::create has the correct signature already